pyproject.toml

```diff
@@ -11,7 +11,7 @@ include = ["mole*"]
 
 [project]
 name = "mole"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
I believe adding a significant new feature should warrant bumping the minor version (e.g., from 0.1 to 0.2) instead of merely a patch release.

Reply:
Agree